R/html_resources.R

```diff
@@ -269,8 +277,7 @@ discover_rmd_resources <- function(rmd_file, encoding,
               recursive = TRUE,
               include.dirs = FALSE)
             lapply(files, function(f) {
-              list(path = file.path(explicit_res$path, f), 
-                   explicit = TRUE,
+              discover_single_resource(file.path(explicit_res$path, f), TRUE,
```

Comment:
Also, a glob could reference a set of both files and directories, correct? Should that recurse? Maybe I need to see a glob test to understand this better. I know it's not the bug you're fixing, but I'm wondering if there's a related issue still here.